src/drizzle/0008_add-admins.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE "admins" (
+	"user_name" varchar(39) PRIMARY KEY NOT NULL,
```

Comment:
From your demo this username should match the github name. Should this be called `github_name`? Can this logic break if we use another oAuth method?